lib/src/parse/stylesheet.dart

```diff
@@ -2367,6 +2367,9 @@ relase. For details, see http://bit.ly/moz-document.
     number += _tryDecimal(allowTrailingDot: scanner.position != start.position);
     number *= _tryExponent();
 
+    // Preserve the sign of -0 by representing it as a double.
+    number = (sign.isNegative && number == 0) ? -0.0 : sign * number;
```

Comment:
Is this necessary? I don't mind having `-0` parse to a normal `0`.

Reply:
We technically need it to parse inputs like `math.div(1, -0)`

Reply:
Could we generate an in-memory `-0` value by doing weird stuff with math instead?

Reply:
Not sure if I'm following. Don't we lose all information about `-0` if we parse it as `0` here, causing `math.div` to be invoked on the wrong input?

Reply:
You're right that we don't generate the floating-point value `-0.0`, but I don't think that's inherently a problem—you can still get that value through arithmetic (such as `-1 * 0.0`). Unless there's a compelling case for a user to be able to write a terse negative zero literal, I think preserving backwards compatibility is more important.

Reply:
Ah, I'm understanding now. Agreed.